[DEBUG   ] confapp.settings_manager                 + SETTINGS: local_settings                    confapp.settings_[DEBUG   ] confapp.settings_manager                 + SETTINGS: pyforms.settings confapp.settings_manager - DEBUG -[DEBUG   ] confapp.settings_manager                 + SETTINGS: h5py._conv - DEBUG - Creating converter from 7 to 5[h5py._conv - DEBUG - Creating converter from 5 to 7
h5py._conv - DEBUG - Creating converter from 7 to 5
h5py._conv - DEBUG - Creating converter from 5 to 7
idtrackerai_app.base_idtrackerai - INFO - TENSORFLOW DEVICES: [_DeviceAttributes(/job:localhost/replica:0/task:0/device:CPU:0, CPU, 268435456, 10766062475036361113), _DeviceAt[INFO    ] idtrackerai_app.base_idtrackerai         TENSORFLOW DEVICES: [_DeviceAttributes(/job:localhos/opt/conda/lib/python3.6/site-packages/pyforms_terminal/basewidget.py - DEBUG - --------- LOADING TERMINAL PARAMS ---------
/opt/conda/lib/python3.6/site-packages/pyforms_terminal/basewidget.py - DEBUG - [                       Session]: [20191116_1039_18-2_L_B_q3]
/opt/conda/lib/python3.6/site-packages/pyforms_terminal/basewidget.py - DEBUG - [                         Video]: [/hps/nobackup/birney/users/ian/MIKK_F0_tracking/split/novel_object/20191[DEBUG   ] /opt/conda/lib//opt/conda/lib/python3.6/site-packages/pyforms_terminal/basewidget.py - DEBUG - [Video file. Note: overwrite the _video parameter defined in the json. Nice to have to execute the applicati[DEBUG   ] /opt/conda/lib/python3.6/s/opt/conda/lib/python3.6/site-packages/pyforms_terminal/basewidget.py - DEBUG - [                     Apply ROI]: [False]
/opt/conda/lib/python3.6/site-packages/pyforms_terminal/basewidget.py - DEBUG - [                           ROI]: [None]
__main__.video - DEBUG - Video object init
__main__.video - DEBUG - Video(open_multiple_files=False)
__main__.py_utils - INFO - Setting MKL library to use single thread
__main__.segmentation_utils - DEBUG - one single video, computing bkg in parallel from single video
__main__.segmentation_utils - DEBUG - Finished parallel loop for bkg subtraction
__main__.py_utils - INFO - Setting MKL library to use multiple threads
/opt/conda/lib/python3.6/site-packages/pyforms_terminal/basewidget.py - DEBUG - [           Subtract background]: [True]
/opt/conda/lib/python3.6/site-packages/pyforms_terminal/basewidget.py - DEBUG - [            Check segmentation]: [False]
/opt/conda/lib/python3.6/site-packages/pyforms_terminal/basewidget.py - DEBUG - [          Resolution reduction]: [1.0]
/opt/conda/lib/python3.6/site-packages/pyforms_terminal/basewidget.py - DEBUG - [          Intensity thresholds]: [[0, 200]]
/opt/conda/lib/python3.6/site-packages/pyforms_terminal/basewidget.py - DEBUG - [               Area thresholds]: [[25, 5000]]
/opt/conda/lib/python3.6/site-packages/pyforms_terminal/basewidget.py - DEBUG - [             Tracking interval]: [[0, 17400]]
/opt/conda/lib/python3.6/site-packages/pyforms_terminal/basewidget.py - DEBUG - [             Number of animals]: [2.0]
/opt/conda/lib/python3.6/site-packages/pyforms_terminal/basewidget.py - DEBUG - [            Tracking intervals]: [None]
/opt/conda/lib/python3.6/site-packages/pyforms_terminal/basewidget.py - DEBUG - [                      Multiple]: [False]
/opt/conda/lib/python3.6/site-packages/pyforms_terminal/basewidget.py - DEBUG - [      Track without identities]: [False]
/opt/conda/lib/python3.6/site-packages/pyforms_terminal/basewidget.py - DEBUG - --------- END LOADING TERMINAL PARAMS ---------
/opt/conda/lib/python3.6/site-packages/pyforms_terminal/basewidget.py - DEBUG - --------- FIELDS FINAL VALUES ---------
/opt/conda/lib/python3.6/site-packages/pyforms_terminal/basewidget.py - DEBUG - [                      _session]: [20191116_1039_18-2_L_B_q3]
/opt/conda/lib/python3.6/site-packages/pyforms_terminal/basewidget.py - DEBUG - [                        _video]: [/hps/nobackup/birney/users/ian/MIKK_F0_tracking/split/novel_object/20191116_1039_18-2_L_B/q3.avi]
/opt/conda/lib/python3.6/site-packages/pyforms_terminal/basewidget.py - DEBUG - [                   _video_path]: [None]
/opt/conda/lib/python3.6/site-packages/pyforms_terminal/basewidget.py - DEBUG - [                     _applyroi]: [False]
/opt/conda/lib/python3.6/site-packages/pyforms_terminal/basewidget.py - DEBUG - [                          _roi]: [None]
/opt/conda/lib/python3.6/site-packages/pyforms_terminal/basewidget.py - DEBUG - [                        _bgsub]: [True]
/opt/conda/lib/python3.6/site-packages/pyforms_terminal/basewidget.py - DEBUG - [                     _chcksegm]: [False]
/opt/conda/lib/python3.6/site-packages/pyforms_terminal/basewidget.py - DEBUG - [                    _resreduct]: [1.0]
/opt/conda/lib/python3.6/site-packages/pyforms_terminal/basewidget.py - DEBUG - [                    _intensity]: [[0, 200]]
/opt/conda/lib/python3.6/site-packages/pyforms_terminal/basewidget.py - DEBUG - [                         _area]: [[25, 5000]]
/opt/conda/lib/python3.6/site-packages/pyforms_terminal/basewidget.py - DEBUG - [                        _range]: [[0, 17400]]
/opt/conda/lib/python3.6/site-packages/pyforms_terminal/basewidget.py - DEBUG - [                       _nblobs]: [2.0]
/opt/conda/lib/python3.6/site-packages/pyforms_terminal/basewidget.py - DEBUG - [                     _progress]: [None]
/opt/conda/lib/python3.6/site-packages/pyforms_terminal/basewidget.py - DEBUG - [                     _rangelst]: [None]
/opt/conda/lib/python3.6/site-packages/pyforms_terminal/basewidget.py - DEBUG - [               _multiple_range]: [False]
/opt/conda/lib/python3.6/site-packages/pyforms_terminal/basewidget.py - DEBUG - [                       _no_ids]: [False]
/opt/conda/lib/python3.6/site-packages/pyforms_terminal/basewidget.py - DEBUG - --------- END FIELDS FINAL VALUES ---------
__main__.video - DEBUG - Video object init
__main__.video - DEBUG - Video(open_multiple_files=False)
__main__.video - INFO - the folder /hps/nobackup/birney/users/ian/MIKK_F0_tracking/split/novel_object/20191116_1039_18-2_L_B/session_20191116_1039_18-2_L_B_q3 has been created
idtrackerai_app.base_idtrackerai - DEBUG - create Chosen_Video
__main__.video - DEBUG - Video object init
__main__.video - DEBUG - Video(open_multiple_files=False)
idtrackerai_app.base_idtrackerai - DEBUG - before init PreprocessingPreviewAPI
idtrackerai_app.base_idtrackerai - DEBUG - pre object: <idtrackerai.preprocessing_preview_api.PreprocessingPreviewAPI object at 0x14bff1723710>
idtrackerai_app.base_idtrackerai - DEBUG - call: init_preview
idtrackerai.preprocessing_preview_api - DEBUG - init_preview
idtrackerai.preprocessing_preview_api - DEBUG - init_preproc_parameters
idtrackerai.preprocessing_preview_api - DEBUG - init_segment_zero
idtrackerai_app.base_idtrackerai - DEBUG - call: init_preproc_parameters
idtrackerai.preprocessing_preview_api - DEBUG - init_preproc_parameters
idtrackerai_app.base_idtrackerai - DEBUG - call: segment
idtrackerai.preprocessing_preview_api - DEBUG - segment
idtrackerai.preprocessing_preview_api - DEBUG - segment 1
idtrackerai.preprocessing_preview_api - DEBUG - segment 2
__main__.video - INFO - saving video object in /hps/nobackup/birney/users/ian/MIKK_F0_tracking/split/novel_object/20191116_1039_18-2_L_B/session_20191116_1039_18-2_L_B_q3/video_object.npy
__main__.video - DEBUG - Video.save(open_multiple_files=False)
idtrackerai_app.base_idtrackerai - DEBUG - call: compute_list_of_blobs
__main__.py_utils - INFO - Setting MKL library to use single thread
__main__.segmentation - INFO - There is only one path, segmenting by frame indices
__main__.segmentation - INFO - An error occurred while reading frame number : 400
__main__.segmentation - INFO - ret: False
__main__.segmentation - INFO - frame: None
__main__.py_utils - INFO - Setting MKL library to use multiple threads
idtrackerai_app.base_idtrackerai - DEBUG - call: check_segmentation_consistency
idtrackerai_app.base_idtrackerai - DEBUG - call: save_list_of_blobs_segmented
__main__.video - INFO - saving video object in /hps/nobackup/birney/users/ian/MIKK_F0_tracking/split/novel_object/20191116_1039_18-2_L_B/session_20191116_1039_18-2_L_B_q3/video_object.npy
__main__.video - DEBUG - Video.save(open_multiple_files=False)
idtrackerai_app.base_idtrackerai - DEBUG - call: model_area_and_crossing_detector
idtrackerai_app.base_idtrackerai - DEBUG - call: train_and_apply_crossing_detector
__main__.crossing_detector - INFO - Discriminating blobs representing individuals from blobs associated to crossings
h5py._conv - DEBUG - Creating converter from 5 to 3
__main__.video - INFO - setting path to save crossing detector model
__main__.crossing_detector - INFO - Get individual and crossing images labelled data
__main__.get_crossing_data_set - DEBUG - number of crossing blobs (in get list): 625
__main__.get_crossing_data_set - DEBUG - number of individual blobs (before cut): 19674
__main__.get_crossing_data_set - INFO - number of sure crossing images used for training: 625
__main__.get_crossing_data_set - INFO - number of individual images used for training: 625
__main__.get_crossing_data_set - INFO - Generating crossing training set.
__main__.get_crossing_data_set - DEBUG - resampling factor crossings: 0.23529411764705882
__main__.get_crossing_data_set - INFO - Generating single individual training set
__main__.get_crossing_data_set - DEBUG - resampling factor individual: 0.23529411764705882
__main__.get_crossing_data_set - INFO - Preparing images and labels
__main__.get_crossing_data_set - INFO - number of sure crossing images used for training: 625
__main__.get_crossing_data_set - INFO - number of individual images used for training: 625
__main__.get_crossing_data_set - INFO - Generating crossing validation set.
__main__.get_crossing_data_set - DEBUG - resampling factor crossings: 0.23529411764705882
__main__.get_crossing_data_set - INFO - Generating single individual validation set
__main__.get_crossing_data_set - DEBUG - resampling factor individual: 0.23529411764705882
__main__.get_crossing_data_set - INFO - Preparing images and labels
__main__.crossing_detector - INFO - Start crossing detector training
__main__.crossing_detector - INFO - Crossing detector training finished
__main__.crossing_detector - INFO - crossing image shape (40, 40, 1)
__main__.crossings_detector_model - INFO - Building graph....
tensorflow - WARNING - From /opt/conda/lib/python3.6/site-packages/tensorflow/python/framework/op_def_library.py:263: colocate_with (from tensorflow.python.framework.ops) is deprecated and will be removed in a future version.
Instructions for updating:
Colocations handled automatically by placer.
tensorflow - WARNING - From /opt/conda/lib/python3.6/site-packages/idtrackerai/utils/cnn_utils.py:192: calling dropout (from tensorflow.python.ops.nn_ops) with keep_prob is deprecated and will be removed in a future version.
Instructions for updating:
Please use `rate` instead of `keep_prob`. Rate should be set to `rate = 1 - keep_prob`.
tensorflow - WARNING - From /opt/conda/lib/python3.6/site-packages/tensorflow/python/ops/losses/losses_impl.py:209: to_float (from tensorflow.python.ops.math_ops) is deprecated and will be removed in a future version.
Instructions for updating:
Use tf.cast instead.
tensorflow - WARNING - From /opt/conda/lib/python3.6/site-packages/idtrackerai/network/crossings_detector_model/crossings_detector_model.py:165: div (from tensorflow.python.ops.math_ops) is deprecated and will be removed in a future version.
Instructions for updating:
Deprecated in favor of operator or tf.math.divide.
__main__.crossings_detector_model - DEBUG - /hps/nobackup/birney/users/ian/MIKK_F0_tracking/split/novel_object/20191116_1039_18-2_L_B/session_20191116_1039_18-2_L_B_q3/crossings_detector/conv already exists
__main__.crossings_detector_model - DEBUG - /hps/nobackup/birney/users/ian/MIKK_F0_tracking/split/novel_object/20191116_1039_18-2_L_B/session_20191116_1039_18-2_L_B_q3/crossings_detector/softmax already exists
__main__.train_crossing_detector - INFO - 
Training Deep Crossing Detector
__main__.train_crossing_detector - DEBUG - entering the epochs loop...
__main__.stop_training_criteria_crossings - INFO - The individual accuracies in validation is 1. for all the classes, we stop the training

__main__.train_crossing_detector - DEBUG - validation losses: [1.8913916e-07, 1.6503337e-07, 4.2355907e-05, 5.0670837e-06, 6.1397004e-06, 4.3088694e-06, 3.947837e-06, 3.5714402e-06, 3.2294267e-06, 2.9048667e-06, 2.6255548e-06]
__main__.crossing_detector - DEBUG - crossing image size 170
__main__.video - INFO - saving video object in /hps/nobackup/birney/users/ian/MIKK_F0_tracking/split/novel_object/20191116_1039_18-2_L_B/session_20191116_1039_18-2_L_B_q3/video_object.npy
__main__.video - DEBUG - Video.save(open_multiple_files=False)
__main__.crossing_detector - DEBUG - Freeing memory. Validation and training crossings sets deleted
__main__.get_crossing_data_set - INFO - number of test images: 104552
__main__.crossing_detector - DEBUG - Classify individuals and crossings
__main__.get_predictions_crossings - DEBUG - There is NOT enough RAM to host 104552 images
__main__.get_crossing_data_set - DEBUG - resampling factor test: 0.23529411764705882
__main__.get_crossing_data_set - DEBUG - resampling factor test: 0.23529411764705882
idtrackerai_app.base_idtrackerai - ERROR - Unable to set dataset extent (dimension cannot exceed the existing maximal size (new: 174001 max: 174000))
Traceback (most recent call last):
  File "/opt/conda/lib/python3.6/site-packages/idtrackerai_app/base_idtrackerai.py", line 214, in track_video
    if self.step1_pre_processing():
  File "/opt/conda/lib/python3.6/site-packages/idtrackerai_app/base_idtrackerai.py", line 347, in step1_pre_processing
    pre.train_and_apply_crossing_detector()
  File "/opt/conda/lib/python3.6/site-packages/idtrackerai/preprocessing_preview_api.py", line 255, in train_and_apply_crossing_detector
    plot_flag=conf.PLOT_CROSSING_DETECTOR,
  File "/opt/conda/lib/python3.6/site-packages/idtrackerai/crossing_detector.py", line 205, in detect_crossings
    blob.set_image_for_identification(video)
  File "/opt/conda/lib/python3.6/site-packages/idtrackerai/blob.py", line 708, in set_image_for_identification
    image_for_identification.shape[1],
  File "/opt/conda/lib/python3.6/site-packages/h5py/_hl/dataset.py", line 625, in resize
    self.id.set_extent(size)
  File "h5py/_objects.pyx", line 54, in h5py._objects.with_phil.wrapper
  File "h5py/_objects.pyx", line 55, in h5py._objects.with_phil.wrapper
  File "h5py/h5d.pyx", line 285, in h5py.h5d.DatasetID.set_extent
RuntimeError: Unable to set dataset extent (dimension cannot exceed the existing maximal size (new: 174001 max: 174000))
ng_data_set           resampling factor test: 0.23529411764705882
[ERROR   ] idtrackerai_app.base_idtrackerai         Unable to set dataset extent (dimension cannot exceed the existing maximal size (new: 174001 max: 174000))
Traceback (most recent call last):
  File "/opt/conda/lib/python3.6/site-packages/idtrackerai_app/base_idtrackerai.py", line 214, in track_video
    if self.step1_pre_processing():
  File "/opt/conda/lib/python3.6/site-packages/idtrackerai_app/base_idtrackerai.py", line 347, in step1_pre_processing
    pre.train_and_apply_crossing_detector()
  File "/opt/conda/lib/python3.6/site-packages/idtrackerai/preprocessing_preview_api.py", line 255, in train_and_apply_crossing_detector
    plot_flag=conf.PLOT_CROSSING_DETECTOR,
  File "/opt/conda/lib/python3.6/site-packages/idtrackerai/crossing_detector.py", line 205, in detect_crossings
    blob.set_image_for_identification(video)
  File "/opt/conda/lib/python3.6/site-packages/idtrackerai/blob.py", line 708, in set_image_for_identification
    image_for_identification.shape[1],
  File "/opt/conda/lib/python3.6/site-packages/h5py/_hl/dataset.py", line 625, in resize
    self.id.set_extent(size)
  File "h5py/_objects.pyx", line 54, in h5py._objects.with_phil.wrapper
  File "h5py/_objects.pyx", line 55, in h5py._objects.with_phil.wrapper
  File "h5py/h5d.pyx", line 285, in h5py.h5d.DatasetID.set_extent
RuntimeError: Unable to set dataset extent (dimension cannot exceed the existing maximal size (new: 174001 max: 174000))
